### DISCUSSION TITLE
Minify css and html

### DISCUSSION THREAD

**Discussion by ArielGueta** [lit/lit#2587] [Q&A]
I'm using terser to minify our code but it doesn't minify the css and html code that's inside the the `html` and `css` lit functions. How we can minify the code?

**Comment by Westbrook** [comment]
We use https://www.npmjs.com/package/rollup-plugin-minify-html-literals to much success.

**Comment by zandaqo** [comment]
Speaking of which, has anyone checked what's the size reduction when compressed?

I vaguely remember mifying lit-html templates years ago with Webpack (if memory serves) and the size reduction was miniscule after compression. It seemed logical to me then, since there wasn't much minification going on aside from whitespace removal and that compresses well being a repetitive pattern.

**Reply by e111077** [reply]
The change I've seen in lit.dev's development is miniscule; biggest code reductions I've seen have been pulling out license comments into their own files.

**Comment by ArielGueta** [comment]
@Westbrook Thanks. Does it work with css?

**Reply by Westbrook** [reply]
There’s some options around that that you can pass to the underlying minifying library, but we minify CSS elsewhere in our toolkit, so I’m not 100% sure. https://www.npmjs.com/package/minify-html-literals

**Reply by ArielGueta** [reply]
>  but we minify CSS elsewhere in our toolkit,

What do you mean?
